internal/manifests/operator/manifests.go

```diff
@@ -5,8 +5,9 @@ import (
 
 	configv1alpha1 "github.com/grafana/tempo-operator/apis/config/v1alpha1"
 	"github.com/grafana/tempo-operator/internal/manifests/operator/prometheus"
-)
+	"go.searchlight.dev/grafana-operator"
```

Comment:
What is this all about?

Reply:
Nevermind, I confused the packages. Going to remove this.